src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -665,11 +666,12 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
     render() {
         // Shared link section props
         const {
+            advancedContentInsightsConfig = {},
```

Comment:
so I don't think we should pass the value of feature flip down to these lower components and we only need the feature flip at the consuming component level. i.e. we could control whether ACI is hidden/shown based on the presence of `onAdvancedContentInsightsToggle`, then we won't need to pass `enabled` in the config.

my thinking is that the only new props we have are:
`isAdvancedContentInsightsChecked` <-- this will control the toggle on/off state
`onAdvancedContentInsightsToggle` <-- this is the onChange of the toggle and will control whether ACI is shown

we could change `showContentInsightsToggle`: https://github.com/box/box-ui-elements/blob/c2dfb7efa2db910eee4f3b2cac99b79f73f667f9/src/features/unified-share-modal/UnifiedShareForm.js#L712
to
```
const showContentInsightsToggle =
    onAdvancedContentInsightsToggle &&
    !isEmailLinkSectionExpanded &&
    !isInviteSectionExpanded &&
    !showCollaboratorList &&
    item?.type === ITEM_TYPE_FILE;
```